packages/core/src/components/navigation/Navbar/Navbar.tsx

```diff
@@ -36,67 +76,71 @@ function Navbar() {
   }
 
   return (
-    <header className={`section ${styles.section} section-navbar`}>
-      <UINavbar scrollDirection={scrollDirection}>
-        <UINavbarHeader>
-          <UINavbarRow className="layout__content">
-            {!searchExpanded && (
-              <>
-                <UIIconButton
-                  data-fs-navbar-button-menu
-                  aria-label="Open Menu"
-                  icon={<UIIcon name="List" width={32} height={32} />}
-                  onClick={openNavbar}
-                />
-                <Link
-                  href="/"
-                  prefetch={false}
-                  aria-label="Go to Faststore home"
-                  title="Go to Faststore home"
-                  data-fs-navbar-logo
-                >
-                  <Logo />
-                </Link>
-              </>
-            )}
+    <UINavbar scrollDirection={scrollDirection}>
+      <UINavbarHeader>
+        <UINavbarRow className="layout__content">
+          {!searchExpanded && (
+            <>
+              <UIIconButton
+                data-fs-navbar-button-menu
+                aria-label="Open Menu"
+                icon={<UIIcon name="List" width={32} height={32} />}
+                onClick={openNavbar}
+              />
+              <Link
+                href="/"
+                aria-label="Go to Faststore home"
+                title="Go to Faststore home"
+                data-fs-navbar-logo
+              >
+                <Logo loading="eager" {...logo} />
```

Comment:
Nice! let's observe if this eager loading affect negatively the LCP from the pages 👀 .